docs/content/getting-started/cpp.md

```diff
@@ -0,0 +1,115 @@
+---
+title: C++ Quick Start
+order: 2
+---
+
+## Installing Rerun
+Before adding Rerun to your application, start by [installing the viewer](installing-viewer.md).
+
+The Rerun C++ SDK depends on an install of the `arrow-cpp` library on your system using.
+If you are using [pixi], you can simply type `pixi global install arrow-cpp`.
+Find more information about other package managers at the official Arrow Apache [install guide](https://arrow.apache.org/install/).
+
+If you're using CMake, you can add the following to your `CMakeLists.txt`:
+
+```cmake
+include(FetchContent)
+FetchContent_Declare(rerun_sdk URL https://github.com/rerun-io/rerun/releases/download/prerelease/rerun_cpp_sdk.zip) # TODO(#3962): update link
+FetchContent_MakeAvailable(rerun_sdk)
+```
+
+This will download a bundle with pre-built Rerun C static libraries for most desktop platforms, all Rerun C++ sources and headers, as well as CMake build instructions for them.
+
+Currently, Rerun SDK works with C++17 or newer:
+```cmake
+set_property(TARGET example PROPERTY CXX_STANDARD 17)
+```
+
+Make sure you link with `rerun_sdk`:
+```cmake
+target_link_libraries(example PRIVATE rerun_sdk)
+```
+
+Combining the above, a minimal self-contained `CMakeLists.txt` looks like:
+```cmake
+project(example LANGUAGES CXX)
+cmake_minimum_required(VERSION 3.16)
+
+add_executable(example main.cpp)
+
+# Download the rerun_sdk
+include(FetchContent)
+FetchContent_Declare(rerun_sdk URL https://github.com/rerun-io/rerun/releases/download/prerelease/rerun_cpp_sdk.zip) # TODO(#3962): update link
+FetchContent_MakeAvailable(rerun_sdk)
+
+# Rerun requires at least C++17, but you can use an even newer version.
+set_property(TARGET example PROPERTY CXX_STANDARD 17)
+
+# Link against rerun_sdk.
+target_link_libraries(example PRIVATE rerun_sdk)
+```
+
+## Logging some data
+Add the following code to your `main.cpp`
+<!-- TODO(#3962): Update Link -->
+(This example also lives in the `rerun` source tree [example](https://github.com/rerun-io/rerun/blob/main/examples/cpp/minimal/main.cpp))
+```cpp
+#include <rerun.hpp>
+#include <rerun/demo_utils.hpp>
+
+using namespace rerun::demo;
+
+int main() {
+    // Create a new `RecordingStream` which sends data over TCP to the viewer process.
+    auto rec = rerun::RecordingStream("rerun_example_cpp");
+    rec.connect().throw_on_failure();
+
+    // Create some data using the `grid` utility function.
+    auto points = grid<rerun::Position3D, float>({-10.f, -10.f, -10.f}, {10.f, 10.f, 10.f}, 10);
+    auto colors = grid<rerun::Color, uint8_t>({0, 0, 0}, {255, 255, 255}, 10);
+
+    // Log the "my_points" entity with our data, using the `Points3D` archetype.
+    rec.log("my_points", rerun::Points3D(points).with_colors(colors).with_radii({0.5f}));
```

Comment:
Not sure if this is the right place for it, but wonder if we should have a section somewhere talking a bit about the builder pattern?

Reply:
we talked about this just the moment you were briefly gone from the call: We want to put "general SDK" things like builder pattern, rerun_c relationship etc. into the API doc. Since we don't have those yet the natural place (and source for future api doc) would be rerun.hpp
At least we don't know any better place for this so far.